libs/ui-watt/src/lib/components/card/watt-card.component.ts

```diff
@@ -40,8 +44,9 @@ import { ChangeDetectionStrategy, Component, HostBinding } from '@angular/core';
   `,
 })
 export class WattCardComponent {
+  @Input() variant: 'watt-solid' | 'watt-elevation' = 'watt-elevation';
```

Comment:
Nitpick: could we remove the `watt` prefix from the input? It seems unnecessary to prefix the input.

Reply:
Did it because the style was called watt-elevation, when i should use it directly without doing variant === elevation use watt-elevation class
Should i rename watt-elevation to elevation? Or leave as is?

Reply:
I still think the classes should be prefixed, however this prefix should just be added in the ‘@hostbinding’, so the consumers of the card component dont have to type the prefix themselves.